### ISSUE TITLE
Add support for firebase AppCheck

### ISSUE THREAD

**Issue by Bark-fa** [thisbejim/Pyrebase#394]
Hi, it would be great if you could add support for firebase [AppCheck](https://firebase.google.com/docs/app-check/) 
By using AppCheck one can protect their backend infrastructure from being abused by attackers/malicious users, however, an engineering problem presents itself, there's no way of enabling AppCheck for a single app, it needs to be enabled project wise, for all apps.

I for example have an android app as the front-end and a python backend and use real-time database, the python code fetched data from a website and updates the firebase real-time Database, Enabling AppCheck for the android app is as simple as adding one line of code, but currently, there doesn't seem to be a way to enable it in python, and since the feature must be enabled project-wise, once I enforce it for Android it's also going to start enforcing it for the python backend web app, and since pyrebase doesn't support this feature any read/write operation would be rejected.

Currently, the only "feasible" solution is to re-write the backend in JavaScript.

**Comment by Neutron-Jimmy** [comment]
Were you able to come up with a solution to this issue? I'm currently dealing with the same problem.

**Comment by CrytoxsDiscord** [comment]
Worry no more, you can use firebase-rest-api python module, then you can edit the database file in your VENV and change headers to headers = {"X-Firebase-Appcheck": "APPCHECK TOKEN HERE"}. Hope this helps.

**Comment by AsifArmanRahman** [comment]
> headers = {"X-Firebase-Appcheck": "APPCHECK TOKEN HERE"}

Is that all it requires? If so, I'll try to implement a way for it to be configured in the library it-self.

**Comment by CrytoxsDiscord** [comment]
> > headers = {"X-Firebase-Appcheck": "APPCHECK TOKEN HERE"}
> 
> Is that all it requires? If so, I'll try to implement a way for it to be configured in the library it-self.

Yes, you need to change all headers in init.py file, and you can use ENV so you dont have to go in init file everytime 😉
